File: library/src/main/java/com/getstream/sdk/chat/model/Mute.java
package com.getstream.sdk.chat.model;

import com.getstream.sdk.chat.rest.User;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Mute {
    @SerializedName("user")
    @Expose
    User user;

    @SerializedName("target")
    @Expose
    User target;

    @SerializedName("created_at")
    @Expose
    String created_at;

    @SerializedName("updated_at")
    @Expose
    String updated_at;

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public User getTarget() {
        return target;
    }

    public void setTarget(User target) {
        this.target = target;
    }

    public String getCreated_at() {
        return created_at;
    }

    public void setCreated_at(String created_at) {
        this.created_at = created_at;
    }

    public String getUpdated_at() {
        return updated_at;
    }

    public void setUpdated_at(String updated_at) {
        this.updated_at = updated_at;
    }
}

File: library/src/main/java/io/getstream/chat/android/client/Mute.kt
package io.getstream.chat.android.client

import com.google.gson.annotations.SerializedName


class Mute {
    @SerializedName("user")
    
    var user: User? = null

    @SerializedName("target")
    
    var target: User? = null

    @SerializedName("created_at")
    
    var created_at: String = ""

    @SerializedName("updated_at")
    
    var updated_at: String = ""

}
